src/flask_app.py

```diff
@@ -27,7 +27,7 @@ def input_data():
 	except Exception ,e:
 		status = 'error'
 		results = 'data format not recoginzed'
-		return jsonify({'status': status, 'error':str(e),'results':'no data in the results'}), 400
+		return jsonify({'error':str(e),'status': status, 'results':"[no data in the results]"}), 400
```

Comment:
This is still wrong format.
* I mean if we have a error, we don't need to set results or set it as an empty array `[]`.
   * results is only available for providing results of grouping, if we didn't group anything, so it's nothing to provide
* above variable `results` is unused